ansible_base/lib/dynamic_config/settings_logic.py

```diff
@@ -10,6 +10,22 @@
 #
```

Comment:
Do we have a plan in place incase someone needs to add a dynamic setting before we fully migrate to dynaconf?

Reply:
I think this will not change, the conditional for the new setting would be added in the same function.

I added as a comment something I want to do for the near future.

```
TODO: In a future implementation this function will defer each section to specific app logic.
    each app will have its own get_defaults function that will return the settings to be merged.
    e.g:
        from ansible_base.resource_registry.settings import get_defaults
        resource_registry_settings = get_defaults(settings)
        dab_data.update(resource_registry_settings)
    The call here can be explicit, or dynamic based on the INSTALLED_APPS list.
```

So ideally, the settings_logic can also go away and each DAB app provide its own `settings.py`

Reply:
TBH, I dont understand the requirement to add a new dynamic setting to the pyproject file